compiler/damlc/daml-stdlib-src/DA/List.daml

```diff
@@ -474,6 +475,11 @@ repeatedly _ [] = []
 repeatedly f as = b :: repeatedly f as'
     where (b, as') = f as
 
+-- | Splits a list into chunks of length @n@. The last chunk will be shorter
+-- than @n@ in case the length of the input is not divisible by @n@.
+chunksOf : Int -> [a] -> [[a]]
+chunksOf n = repeatedly (splitAt n)
```

Comment:
in Haskell `repeatedly (splitAt 0) xs` and `repeatedly (splitAt (-1)) xs` would give an infinite list of empty lists (unless `null xs`) but since Daml is strict it would just get stuck, so I think it's better to fail upfront.

```suggestion
chunksOf n | n <= 0 = error "chunksOf: chunk size must be greater than zero, got " <> show n
chunksOf n = repeatedly (splitAt n)
```